nixos/modules/services/web-apps/freshrss.nix

```diff
@@ -0,0 +1,247 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.my.services.freshrss;
+
+  poolName = "freshrss";
+in
+{
+  meta.maintainers = with maintainers; [ etu stunkymonkey ];
+
+  options.my.services.freshrss = {
+    enable = mkEnableOption "FreshRSS feed reader";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.freshrss;
+      defaultText = lib.literalExpression "pkgs.freshrss";
+      description = "Which FreshRSS package to use.";
+    };
+
+    defaultUser = mkOption {
+      type = types.str;
+      default = "admin";
+      description = "Default username for FreshRSS.";
+      example = "eva";
+    };
+
+    passwordFile = mkOption {
+      type = types.str;
+      description = "Password for the defaultUser for FreshRSS.";
+      example = "/run/secrets/freshrss";
+    };
+
+    baseUrl = mkOption {
+      type = types.str;
+      description = "Default URL for FreshRSS.";
+      example = "https://freshrss.example.com";
+    };
```

Comment:
These two options are missing `default`.

Reply:
The intention is the sysadmin is required to fill in these values. There should be no default.